grunt_tasks/localConfig.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Kitware Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs');
+
+module.exports = function (grunt) {
+    const confDir = `${grunt.config.get('girderDir')}/conf`;
+    const localConfigPath = `${confDir}/girder.local.cfg`;
+
+    grunt.config.merge({
+        copy: {
+            'local-config': {
+                src: `${confDir}/girder.dist.cfg`,
+                dest: localConfigPath
+            }
```

Comment:
Yeah, I was really surprised to see it here too, and agree that it should live in the Python code.

However, this PR is just a refactor / bugfix of some of existing functionality, so moving the functionality is a bit out of scope. I've created an issue here: https://github.com/girder/girder/issues/2263 , which may be good for @sutartmelson or @Pierre-Assemat to handle.